internal/server/server.go

```diff
@@ -204,6 +205,10 @@ func initRuleset() error {
 		}
 		conf.Log.Infof("start to initialize ruleset")
 		_, counts, err := rulesetProcessor.Import(content)
+		if err != nil {
```

Comment:
@superrxan Is this expected to return for error? Moreover, suggest to use conf.Log.Errorf instead of Infof

Reply:
From my perspective, if we don't return for error, there could be an out-of-index panic when the later log calls for counts[x].

Reply:
I agree with return for error